_archiv/2023/registrierung.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: program
+title: "Registrierung ESS 2023"
+registrationstart: "2023-03-01T00:00:00+02:00"
+deadline: "2023-05-29T23:59:59+02:00"
+---
+
+<div class="row justify-content-center">
+
+    <div class="col-12">
+
+        <p>Die Registrierung ist vom 01. März bis zum 29. Mai 2023 (23:59 Uhr, UTC+2) geöffnet.<br/>
```

Comment:
Vielleicht auch wieder die Variablen nutzen?
```suggestion
        <p>Die Registrierung ist vom {% include date-german.html date=page.registrationstart
            showDayOfWeek=false showDay=true %}
            {{ formattedDate }} bis zum {% include date-german.html
            date=page.deadline
            showDayOfWeek=false showDay=true %}
            {{ formattedDate }} (23:59 Uhr, UTC+2) geöffnet.<br/>
```